### ISSUE TITLE
[Feature] Show activity reminder only after goal time reached

### ISSUE THREAD

**Issue by Geomitron** [Razeeman/Android-SimpleTimeTracker#116]
I'm using "goal time" as an ideal maximum time for certain task "sessions", and I want notifications to constantly pester me until I switch to another task. But I only want them to start after my session goal time has been reached.

When "Show activity reminder" is enabled, I would like an additional option to appear below the "Recurrent" option to only have the reminder show after the activity's session goal time has been reached. Some potential option descriptions are "Only show after session goal" or "Wait until session goal".

This seems like a fairly trivial option to add; I'm familiar enough with writing Kotlin where I could attempt to create a pull request for this, although I'm sure it would take @Razeeman much less time to add.

This app has been very helpful for my productivity, and it's very nice to use. Thanks!

**Comment by Geomitron** [comment]
Alternatively, it may be useful to make this another "reminder" section entirely, so that activity reminder notifications can happen at a different rate depending on if it's before or after the session goal
